docs/server/source/cloud-deployment-templates/revoke-tls-certificate.rst

```diff
@@ -0,0 +1,40 @@
+How to Revoke an SSL/TLS Certificate
+====================================
+
+This page enumerates the steps *we* take to revoke a self-signed SSL/TLS certificate 
+in a cluster.
+It can only be done by someone with access to the self-signed CA
+associated with the cluster's managing organization.
+
+Step 1: Revoke a Certificate
+----------------------------
+
+Since we used Easy-RSA version 3 to
+:ref:`set up the CA <How to Set Up a Self-Signed Certificate Authority>`,
+we use it to revoke certificates too.
+
+Go to the following directory (associated with the self-signed CA):
+``.../bdb-cluster-ca/easy-rsa-3.0.1/easyrsa3``.
+You need to be aware of the base filename used to import the certificate, and
```

Comment:
The base filename is the name used to import the certificate using the `easyrsa import-req` command. Will mention it in the docs.